portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
curious, why we have this comment for Comment class but not for others in the same package?

Reply:
This was the class that was provided in the resource so I am not too sure.